### PR TITLE
chore(api,cli,pkg): remove deprecated function on validator package

### PR DESCRIPTION
**Pull request by henrybarreto**

### CODE REVIEW

File: pkg/models/user.go [hunk 1]
```diff
@@ -13,7 +15,45 @@ type UserData struct {
 }
 
 type UserPassword struct {
-	Password string `json:"password" bson:",omitempty" validate:"required,min=5,max=30"`
+	// Plain contains the plain text password.
+	Plain string `json:"password" bson:"-" validate:"required,password"`
+	// Password contains the hashed pasword from plain text.
+	Password string `json:"-" bson:"password"`
```
Comment (heiytor) [inline pkg/models/user.go:21]: Isn't it a good idea to call this 'Hash' instead of 'Password'?

Comment (otavio) [reply pkg/models/user.go:21]: `HashedPassword`?

Comment (henrybarreto) [reply pkg/models/user.go]: Done. It is now `PlainPassword` and `HashedPassword`.

File: api/pkg/echo/handlers/validator.go [hunk 2]
```diff
@@ -18,13 +18,8 @@ func NewValidator() *Validator {
...
 func (v *Validator) Validate(structure interface{}) error {
 	// Use the ShellHub package validator to validate the request body.
-	if ok, err := v.validator.Struct(structure); !ok {
-		fields, err := validator.GetInvalidFieldsFromErr(structure, err)
-		if err != nil {
-			return err
-		}
-
-		return errors.NewErrInvalidEntity(fields)
+	if ok, err := v.validator.Struct(structure); !ok || err != nil {
```
Comment (otavio) [inline api/pkg/echo/handlers/validator.go:21]: The word 'deprecated' has been mistyped.

**Pull request merged**